modules/dnn/test/test_tf_importer.cpp

```diff
@@ -40,7 +40,7 @@ TEST(Test_TensorFlow, read_inception)
     ASSERT_TRUE(!sample.empty());
     Mat input;
     resize(sample, input, Size(224, 224));
-    input -= 128; // mean sub
+    input -= Scalar(128); // mean sub
```

Comment:
@dkurt, `Scalar::all(128)` ?

BTW, Test doesn't test inference result, only loading is tested here.

Reply:
Actually, I'm not sure about mean values for this model.

Reply:
`Scalar(128)` is similar to `Scalar(128, 0, 0, 0)`
`Scalar::all(128)` is `Scalar(128, 128, 128, 128)`

Reply:
Yea, I just nor sure if we really need to pass [128, 128, 128] or [0, 0, 0]. From the origin repo, they use different value:
https://github.com/petewarden/tf_ios_makefile_example/blob/880c92d2a15d65b32319d43cca5987481069b574/RunModelViewController.mm#L175L209.
So I think we can replace it even to `Scalar::all(117)`.